Makefile

```diff
@@ -68,13 +67,15 @@ endif
 REPORTS = $(ROOT_DIR)/reports
 TEST_PUBLISH_FLAGS = --junitfile-testsuite-name=relative --junitfile-testcase-classname=relative --junitfile $(REPORTS)/unittest.xml
 
+.EXPORT_ALL_VARIABLES:
+
 all: build
 
 ci-lint:
 	${ROOT_DIR}/tools/check-commits.sh
 	${ROOT_DIR}/tools/handle_ocp_versions.py
-	$(MAKE) verify-latest-onprem-config
-	$(MAKE) validate-swagger-changes
+	skipper $(MAKE) generate-all
+	git diff --exit-code  # this will fail if changes have not been committed
```

Comment:
Great, maybe change the comment to `# this will fail if generate-all caused any diff`